### PR TITLE
`winit_wasm` requires `render` feature

### PR DESCRIPTION
**Pull request by dpc**
Otherwise it will throw an exception on start.

### CODE REVIEW

**Comment by bjorn3** [comment]
Can you please also update https://github.com/bevyengine/bevy/blob/bc499591c2d090e17c95d2c59bc6e16ba60a023d/Cargo.toml#L516?

**Review by mockersf — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -503,7 +503,7 @@ required-features = []
 [[example]]
 name = "assets_wasm"
 path = "examples/wasm/assets_wasm.rs"
-required-features = ["bevy_winit"]
+required-features = ["bevy_winit","render"]
```
Comment (mockersf) [inline Cargo.toml:506]: This line is not for the `winit_wasm` example, it's for `assets_wasm`

Could you update the one ten lines down?

**Review by mockersf — CHANGES_REQUESTED** [review]
this

**Review by mockersf — APPROVED** [review]

**Comment by cart** [comment]
`bevy_winit` and `render` are both default features. I'd like us to move away from specifying default features for examples, unless they are meant to be run without default features. I think what really needs to change is our `examples/README.md` telling people to run wasm examples without default features.

**Comment by mockersf** [comment]
the wasm examples have been removed as they are no longer relevant, most examples should work in wasm with default features now 👍

**Comment by dpc** [comment]
I guess no longer needed then

**Pull request closed without merging**